charts/vald/templates/jobs/db/initialize/scylla/configmap.yaml

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
Maybe you committed these files in initialize/scylla directory accidentally.

Reply:
The reason I committed the initialize/scylla template was because scylladb needed to freeze the secondary index, so I changed the configmaps.
I thought it would be lower cost to output this change to k8s/jobs/initialize/scylla than to reflect it in cassandra

Reply:
I see.
Then, please define a new `initializer.scylla` section in values.yaml.

Reply:
okay, I'll work on it.

Reply:
Although there are no differences other than configmap queries, do I still need to prepare a values.yaml section for scylla?
Currently, when using cassandra backup/meta, the cassandra configuration takes precedence, so I would like to use the cassandra configuration as the value for values.yaml's scylla config.

Reply:
You should.
Because with the current implementation, if we enable `initializer.cassandra.enabled`, the two jobs, cassandra-init and scylla-init will run by executing `helm install`. So please, at least, define `initializer.scylla.enabled` or something like that. The other configurations are okay to share with the cassandra section, I think.

Reply:
hmm, I checked cassandra query specs, they also recommend frozen list for secondary index.
I'll remove scylla chart templates and change cassandra configmap to use frozen list

Reply:
okay :+1:

Reply:
but we have to care about out test environment cassandra, could you please cooperate about test them?

Reply:
sure.